src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -108,27 +114,12 @@ public function update( $node_id, $element, WPML_PB_String $string ) {
 	}
 
 	/**
-	 * @param array  $element
-	 * @param string $field_key
+	 * @param string $field
 	 *
-	 * @return bool
-	 */
-	private function is_flat_field( $element, $field_key ) {
-		return isset( $element[ self::SETTINGS_FIELD ][ $field_key ] )
-		       && is_string( $element[ self::SETTINGS_FIELD ][ $field_key ] )
-		       && trim( $element[ self::SETTINGS_FIELD ][ $field_key ] );
-	}
-
-	/**
-	 * @param array      $element
-	 * @param string|int $field_wrapper
-	 * @param string     $field_key
-	 *
-	 * @return bool
+	 * @return string[]
 	 */
-	private function is_array_field( $element, $field_wrapper, $field_key ) {
-		return isset( $element[ self::SETTINGS_FIELD ][ $field_wrapper ][ $field_key ] )
-		       && trim( $element[ self::SETTINGS_FIELD ][ $field_wrapper ][ $field_key ] );
+	private static function get_partial_path( $field ) {
+		return explode( '|', $field );
```

Comment:
@strategio Would `>` work? eg `foo>bar`? I'm not sure if this would cause an issue in wpml_config.xml.

Reply:
Yes @brucepearson it works with `>`. I tested this in WPML > Settings > Custom XML Config, no problem. And string registration, data translation works perfectly. I changed it!